generator/template/process.go

```diff
@@ -61,6 +63,52 @@ func processSQLBuilder(dirPath string, dialect jet.Dialect, schemaMetaData metad
 	processTableSQLBuilder("table", sqlBuilderPath, dialect, schemaMetaData, schemaMetaData.TablesMetaData, sqlBuilderTemplate)
 	processTableSQLBuilder("view", sqlBuilderPath, dialect, schemaMetaData, schemaMetaData.ViewsMetaData, sqlBuilderTemplate)
 	processEnumSQLBuilder(sqlBuilderPath, dialect, schemaMetaData.EnumsMetaData, sqlBuilderTemplate)
+	processTableSQLBuilderSetSchema(sqlBuilderPath, schemaMetaData, sqlBuilderTemplate)
+}
+
+func processTableSQLBuilderSetSchema(dirPath string, schemaMetadata metadata.Schema, builderTemplate SQLBuilder) {
+
+	var builders []TableSQLBuilder
+	for _, tm := range schemaMetadata.TablesMetaData {
+
+		table := builderTemplate.Table(tm)
+		if table.Skip {
+			continue
+		}
+
+		builders = append(builders, table)
+	}
+
+	if len(builders) == 0 {
+		return
+	}
+
+	err := utils.EnsureDirPath(dirPath)
+	throw.OnError(err)
+
+	fmt.Println("Generating global `SetSchema` method")
+	schemaIdentifier := utils.ToGoIdentifier(schemaMetadata.Name)
+
+	funcPath := path.Join(dirPath, builders[0].Path)
```

Comment:
file can be  named just `table.go` and `view.go`. You can reuse `fileTypes` inside `processTableSQLBuilder`.

Reply:
Updated in 7db99b1